apps/console/src/index.jsp

```diff
@@ -276,6 +276,38 @@
                     return applicationDomain.replace(/\/+$/, '') + getTenantPath();
                 }
 
+                /**
+                 * Construct the auth params for organization login `authorize` requets.
+                 *
+                 * @remarks This only applies to the new authz runtime.
+                 *
+                 * @returns {string} Contructed auth params.
+                 */
+                function getAuthParamsForOrganizationLogins(orginalPrams) {
+                    if (startupConfig.legacyAuthzRuntime) {
```

Comment:
```suggestion
                    if (startupConfig?.legacyAuthzRuntime) {
```

Reply:
`?` would only work in latest browsers. anywaty `startupConfig` should be defined before executing the preauth logic.